### PR TITLE
Issue/82/unlensed agn instance catalogs

### PR DESCRIPTION
**Pull request by jbkalmbach**
This pull request is to enable AGN variability in instance catalogs with the new multiband tau. Instead of changing sims_catUtils and potentially breaking things for other uses there, I just took the code we needed out of sims_catutils and added it here with the necessary changes. This also adds another flag to `generateInstCat.py` to turn on/off AGN in instance catalogs.

### CODE REVIEW

**Comment by jbkalmbach** [comment]
Successfully tested on NERSC with shifter image `lsstdesc/stack-sims-cat:w_2019_42-sims_w_2019_42` running command: `python generateInstCat.py --config_file ../workspace/run2.1/test_agn_config_file_2.1.ddf.json --fov 0.05 --ids 2500000 --agn_threads 2`

File: workspace/run2.1/test_agn_config_file_2.1.ddf.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{"fov": 0.1,
+"unlensed_agn": 1,
+"gzip_threads": 1,
+"descqa_catalog": "cosmoDC2_v1.1.4_image_addon_knots",
+"sed_lookup_dir": "/global/projecta/projectdirs/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4/sedLookup/",
+"db": "/global/projecta/projectdirs/lsst/groups/SSim/DC2/DDF_cadences/ddf_cadence.db",
+"agn_db_name": "/global/cscratch1/sd/jchiang8/desc/Run2.2i/agn_catalog/2019-11-21/cosmoDC2_agn.db",
```
Comment (jchiang87) [inline workspace/run2.1/test_agn_config_file_2.1.ddf.json]: We should move this out of my scratch area and into some place like `/global/projecta/projectdirs/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4` and rename the file appropriately.  I created that file using the code that's currently in master via #80 .

**Review by jchiang87 — CHANGES_REQUESTED** [review]
The code looks ok to me, but I'm not familiar enough with the catsim code to do a real review of  `Variability.py` module.  We should put the `cosmoDC2_agn.db` file in a more permanent place and update the test agn config file path.

File: workspace/run2.1/test_agn_config_file_2.1.ddf.json (hunk 1 above)
Comment (jchiang87) [reply workspace/run2.1/test_agn_config_file_2.1.ddf.json]: I've copied that file to 
```
/global/projecta/projectdirs/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4/agn_cosmoDC2_v1.1.4.db
```

**Comment by jbkalmbach** [comment]
Ok, I updated the location for the AGN database. I also submitted a [related PR in `sims_catUtils`](https://github.com/lsst/sims_catUtils/pull/106) to stop the `None` files from appearing when running instance catalog generation.

**Review by jchiang87 — APPROVED** [review]
lgtm.

**Comment by jbkalmbach** [comment]
If nobody has any other comments, I will merge this at 4pm Pacific this afternoon.

**Pull request merged**